src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -338,7 +338,12 @@ private int declareNamespace(String namespace, TypedVar symbol, boolean isDefaul
     emitNamespaceBegin(namespace);
     TreeWalker treeWalker = new TreeWalker(compiler.getTypeRegistry(), provides);
     if (isDefault) {
-      treeWalker.walk(symbol);
+      if (isPrivate(symbol.getJSDocInfo())) {
+        emit("// skipping emitting private default export " + symbol.getName() + ".");
```

Comment:
as discussed offline, this is because we generate default module export for this symbol.